### PR TITLE
#167727721 Setup api documentation

### PR DESCRIPTION
**Pull request by kola-1**
#### What this PR does
This PR will setup API documentation with swagger

#### Tasks completed
- install swagger-ui-express
- install swagger-jsdoc
- Setup swagger for documenting API's

#### How this can be manually tested
- Clone this branch
- Change directory into it
- Run `npm install` in your terminal to install all the dependencies
- Run `npm run start:dev` in your terminal to start the development server
- Navigate to this URL `http://localhost:3000/api/v1/docs` from your browser

#### What are the relevant pivotal tracker stories?
[#167727721](https://www.pivotaltracker.com/story/show/167727721)

### CODE REVIEW

File: src/config/swaggerDoc.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import swaggerJsDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+import _ from './env';
+
+const swaggerDefinition = {
+openapi: "3.0.1",
```
[the remark this answers is not in the document]
Comment (tobslob) [reply src/config/swaggerDoc.js]: Nice work, check if you have the latest commits from develop, seems your eslint is not working to make the quotations regular and the ``` import  _  from './env``` should work directly with ``` import './env' ``

File: src/index.js [hunk 2]
```diff
@@ -2,14 +2,26 @@ import express from 'express';
...
 const infoLog = debug('http:info');
+const router = express.Router();
+
+// Pass router to routes
+routes(router);
+
+// Pass router to swagger middleware
+swaggerDoc(router);
+
+app.use('/api', swaggerUi.serve, router);
```
Comment (tejiri4) [inline src/index.js]: Let add version  one to the router here '/api/v1'

File: src/routes/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import response from '../utils/response';
+
+const routes = (router) => {
+  router
+    .route('/')
+  /**
+   * @swagger
+   * /:
+   *   get:
+   *     description: Welcome message
```
Comment (tejiri4) [inline src/routes/index.js]: add more documentation on how the response format looks like

File: src/config/swaggerDoc.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kola-1) [reply src/config/swaggerDoc.js]: Done. Thanks for the feedback

File: src/index.js (hunk 2 above)
Comment (kola-1) [reply src/index.js]: Done

File: src/routes/index.js (hunk 3 above)
Comment (kola-1) [reply src/routes/index.js]: Done. Please check now

**Review by tejiri4 — APPROVED** [review]
LGTM Nice work done

**Pull request merged**